Procedures/Affiliated-Package-Review-Process.md

```diff
@@ -138,7 +138,7 @@ author has a month to respond to any of the points in the review,
 and the coordinator may decide to change the decision.
 
 If the package is accepted, the coordinator edits the pull request
-to add the results of the review to the JSON file.  <!--Once the
+to add the results of the review to the README.md file.  <!--Once the
 continuous integration passes, the pull request is merged.--> The
 pull request is then merged.
```

Comment:
What's with this commented out sentence?

Reply:
It was in the parent Astropy doc.  I commented it out because I don't know that we have that set up, but maybe I should just delete it entirely?  However, now it confuses me because the only PR that is happening is on our README file, and what CI would happen there?  Oh, maybe they had a validator running on their .json file and that's the CI they're talking about?  Not sure.